service/src/application.rs

```diff
@@ -45,8 +45,8 @@ pub struct FcserviceApp {
 impl Default for FcserviceApp {
     fn default() -> Self {
         Self {
-            config: None,
```

Comment:
Not sure if this logic was intentional. A default instance is now being used to avoid unwraps all over the `config` method.

Reply:
I think this is actually coming from the abcissa template.

Reply:
Odd. The template is using an option and the trait always expects a valid reference.
Not sure if `config` was bean  optional value in our case. Meanwhile, it is possible to use the default instance as a dummy placeholder.